icon_validator/workflow/unmarshal.py

```diff
@@ -16,5 +15,6 @@ def read_workflow(spec: dict, file_name: str) -> Workflow:
             wf = json.load(workflow_file)
         except Exception as e:
             raise Exception('Unable to read workflow', e)
-        return from_dict(data_class=Workflow, data=wf)
+        return parse_obj_as(Workflow, wf)
```

Comment:
Is it possible for this to throw exception?

Reply:
reference for parse_obj_as didn't specify an error as referenced here  https://github.com/samuelcolvin/pydantic/blob/master/pydantic/tools.py#L33 but I can put the parse_obj_as within the try block